$NetBSD: patch-synah,v 1.1 2007/05/23 11:44:36 rillig Exp $

These macros are used as lvalues, but were defined as rvalues. The type
casts are unnecessary as well.

--- syna.h.orig	2001-04-26 20:39:58.000000000 +0200
+++ syna.h	2007-05-23 13:41:55.000000000 +0200
@@ -52,9 +52,9 @@
 
 /* core */
 extern guint16 *synx_output[3];
-#define output ((guchar*)(synx_output[0]))
-#define lastOutput ((guchar*)(synx_output[1]))
-#define lastLastOutput ((guchar*)(synx_output[2]))
+#define output (synx_output[0])
+#define lastOutput (synx_output[1])
+#define lastLastOutput (synx_output[2])
 
 #define fadeMode (synx_cfg.fadeMode)
 #define starSize (synx_cfg.starSize)
